### PR TITLE
chore(deps): update github-issues plugin version

### PR DESCRIPTION
**Pull request by dzemanov**
## Description

Update @backstage-community/plugin-github-issues version.

## Which issue(s) does this PR fix

- Fixes [RHIDP-2727](https://issues.redhat.com/browse/RHIDP-2727)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

## How to test changes / Special notes to the reviewer
Community plugins backstage version is: v1.30.2, so maybe this needs to wait till backstage update. Works locally fine:
Screenshot from 2024-10-04 18-24-54

### CODE REVIEW

File: dynamic-plugins/wrappers/backstage-community-plugin-github-issues/package.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
   },
   "backstage": {
     "role": "frontend-plugin",
-    "supported-versions": "1.27.7",
+    "supported-versions": "1.29.2",
```
Comment (dzemanov) [inline dynamic-plugins/wrappers/backstage-community-plugin-github-issues/package.json:14]: Is this version alright, regarding https://github.com/backstage/community-plugins/blob/main/workspaces/github-issues/plugins/github-issues/CHANGELOG.md#044?

**Review by PatAKnight — COMMENTED** [review]
Tested and it worked for me, just have to wait till the main branch gets fixed

**Comment by dzemanov** [comment]
Closing as `github-issues` version was updated in https://github.com/janus-idp/backstage-showcase/pull/1818

**Pull request closed without merging**